### PR TITLE
[GH-6] feature(unit-test): add jest, preact testing library and jest dom

### PR DESCRIPTION
**Pull request by kcubero27**
- [ ] Uncomment TailwindCSS

### CODE REVIEW

**Comment by kcubero27** [comment]
This branch is taking changes from GH-5

File: package.json [hunk 1]
```diff
@@ -5,28 +5,47 @@
...
     "@typescript-eslint/eslint-plugin": "^3.3.0",
     "@typescript-eslint/parser": "^3.3.0",
+    "awesome-typescript-loader": "^5.2.1",
+    "babel-loader": "^8.1.0",
     "eslint": "^7.2.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.20.0",
     "husky": "^4.2.5",
+    "jest": "^26.1.0",
```
Comment (jesugmz) [inline package.json:38]: :heart:

File: src/styles/index.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/*
```
Comment (kcubero27) [inline src/styles/index.css:1]: I should not comment this!!!

File: .storybook/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      },
+      {
+        loader: require.resolve("babel-loader"),
+      },
+    ],
+  });
+
+  config.resolve.extensions.push(".ts", ".tsx");
+
+  // console.log(util.inspect(config, false, null, true));
```
Comment (tomtobac) [inline .storybook/webpack.config.js:47]:
```suggestion
```

**Pull request merged**